vitest.config.ts

```diff
@@ -0,0 +1,20 @@
+import react from '@vitejs/plugin-react';
+
+import { defineConfig } from 'vitest/config';
+
+// https://vitejs.dev/config/
+export default defineConfig({
+  plugins: [react()],
+  test: {
+    environment: 'jsdom',
```

Comment:
Why not use `happy-dom` ? It is faster than jsdom and has all the features we need.